resources/js/store/index.ts

```diff
@@ -0,0 +1,40 @@
+import Vue from 'vue';
+import Vuex, {Store} from 'vuex';
+import { Inertia } from '@inertiajs/inertia';
+import mutations from './mutations';
+
+export function getInitialState() {
```

Comment:
Nice one, we can use RootState in here as well.

Reply:
But RootState is just the types. getInitialState returns the initial values. I think they are different things.

Reply:
Yes, I mean that we should annotate the return type of this function with RootState, that's what it returns after all. No?

Reply:
yes! Excuse moi, I misunderstood your comment :)

Reply:
Annotated!

Reply:
np, probably wasn't clear enough. Thx!